jax/experimental/jax2tf/tests/call_tf_test.py

```diff
@@ -65,6 +66,34 @@ def _named_test(**kwargs):
 
 class CallTfTest(tf_test_util.JaxToTfTestCase):
 
+  @classmethod
+  def setUpClass(cls):
+    # Pick one device from each available platform
+    cls.jax_platforms = []
```

Comment:
It looks like this can be a local variable -- `jax_platforms` is not used by existing test methods.

Reply:
Indeed, moved to the test that needs it (test_multi_platform)